### DISCUSSION TITLE
recordPath on Windows 10 PC how it's should looks like?

### DISCUSSION THREAD

**Discussion by AndriiDorofeiev** [bluenviron/mediamtx#2664] [Questions]
### Question

Hi everybody,

i just started MediaMTX on Windows machine, i want to use video recording, i want to be sure how variable recordPath should look like. Is it right H:\Balter\%Y-%m-%d_%H-%M-%S-%f  ? Or should be translated/adopted some how on windows?

Thank you and best regards,
Andrii

**Comment by aler9** [comment]
Hello, i think you're missing a slash:

```
H:\Balter\%Y-%m-%d_%H-%M-%S-%f
```

Anyway no particular precaution should be taken on Windows, all features are designed to be independent from the operating system.

**Comment by AndriiDorofeiev** [comment]
> ### Question
> Hi everybody,
> 
> i just started MediaMTX on Windows machine, i want to use video recording, i want to be sure how variable recordPath should look like. Is it right H:\Balter%Y-%m-%d_%H-%M-%S-%f ? Or should be translated/adopted some how on windows?
> 
> Thank you and best regards, Andrii

Hi! For those who have the same issue, I found right solution. On Windows it's should be like this: 
```
H:\Balter\%%Y%-%m%-%d%_%H%-%M%-%S%-%f%
```
My script work without dashes 
```
"H:\Balter\%%Y%%m%%d%%H%%M%%S%.mkv"
```

[a comment by an automated account is not shown]
